samples/csharp_dotnetcore/21.corebot-app-insights/CoreBot-App-Insights.csproj

```diff
@@ -15,12 +15,12 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
-    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.8.0" />
-    <PackageReference Include="Microsoft.Recognizers.Text.DataTypes.TimexExpression" Version="1.1.5" />
+    <PackageReference Include="Microsoft.Bot.Builder.AI.Luis" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.ApplicationInsights" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Dialogs" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.ApplicationInsights.Core" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Bot.Builder.Integration.AspNet.Core" Version="4.9.0" />
+    <PackageReference Include="Microsoft.Recognizers.Text.DataTypes.TimexExpression" Version="1.3.2" />
   </ItemGroup>
```

Comment:
This explicit dependency can be removed, DataTypes.TimexExpression is referenced by AdaptiveExpressions that is referenced by AI.Luis.